src/chat/_chatwidget.py

```diff
@@ -132,6 +132,10 @@ def addChannel(self, name, channel, index = None):
         else:
             self.insertTab(index, self.channels[name], name)
 
+    def sortChannels(self):
+        for _, channel in self.channels.items():
```

Comment:
`for channel in self.channels.values():`

Reply:
Well, I took that line from the function 'updateChannels' right below it, why is it ok there and not here? It sure looks better.

Reply:
and while we're on it, should we 'unhide' _sortChatters to sortChatters? Or is there a downside?

Reply:
That line below is some older code, and it's less elegant IMO - I think 'pick all the values' is more straightforward than 'pick all the key-value pairs, then discard the keys'. As for unhiding _sortChatters, we could do it, sure.